### DISCUSSION TITLE
web interface running super slow

### DISCUSSION THREAD

**Discussion by dwxt** [mcguirepr89/BirdNET-Pi#1190] [Q&A]
I don't know if anyone here have same question? i put my pi at yard and when I back home and plug the pi to my router and trying to see result, the web interface running super slow. but when i ssh to the pi, it's fast enough. Need for help. thank you

**Comment by ericatkins** [comment]
How long have you had your install running?  I had this problem and a reinstall sped up performance on my rPi 3B+.  I suspect I did not follow the install instructions accurately for my device on the first install.  _

**Reply by dwxt** [reply]
Since very first... it runs super slow while accessing via local network. I assume maybe any wrong configuration or necessary optimisation I forgot?
